### PR TITLE
update depgroup liberty didn't copy

### PR DESCRIPTION
**Pull request by prb112**
Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: fhir-server-webapp/src/main/liberty/config/server.xml [hunk 1]
```diff
@@ -51,7 +51,7 @@
         
         To send the trace messages to standard out, set TRACE_FILE to "stdout".
     -->
-    <variable name="TRACE_SPEC" defaultValue="*=info"/>
+    <variable name="TRACE_SPEC" defaultValue="*=info:com.ibm.fhir.*=FINE"/>
```
Comment (prb112) [inline fhir-server-webapp/src/main/liberty/config/server.xml:54]:
```suggestion
    <variable name="TRACE_SPEC" defaultValue="*=info"/>
```

File: fhir-server-webapp/pom.xml [hunk 2]
```diff
@@ -104,15 +104,27 @@
...
                 <configuration>
-                    <copyDependencies>
-                        <dependency>
-                            <groupId>org.apache.derby</groupId>
-                            <artifactId>derby</artifactId>
-                        </dependency>
-                        <location>${project.build.directory}/liberty/wlp/usr/shared/resources/lib/derby/</location>
-                    </copyDependencies>
                     <copyDependencies>
                         <!-- Used to Test Profiles and Operations -->
```
Comment (prb112) [inline fhir-server-webapp/pom.xml:108]:
```suggestion
```

**Review by lmsurpre — APPROVED** [review]
I guess the derby change is related to #2477 which upgrades us to derby 10.15.
Assuming it doesn't hurt anything to have those listed here early, this looks good to me.

**Comment by prb112** [comment]
> I guess the derby change is related to #2477 which upgrades us to derby 10.15.
> Assuming it doesn't hurt anything to have those listed here early, this looks good to me.

Exactly

**Pull request merged**